doc/_templates/customindex.html

```diff
@@ -38,13 +38,21 @@ <h2>Open Standard</h2>
     </div>
   </div>
   <div>
-    <h2>What is Autocrypt? What is the status?</h2>
+    <h2>What is Autocrypt?</h2>
     <p>
-      Autocrypt is a specification which mail application programmers can implement
-      to offer their users convenient end-to-end encryption for e-mail. The first specification
-      was released in December 2017 and a few mail programs already have or are heading 
-      towards Autocrypt-capable releases in spring 2018.  Please join us in implementing
-      and refining "Level 1" and other upcoming specifications. For any further questions
+      Autocrypt is a specification for convenient
+      end-to-end-encryption of E-Mails. Mail application
+      programmers are currently implementing it.
```

Comment:
"Mail application programmers" is a little bit odd.  how about "Developers are working on adding it to several different e-mail programs."?